src/program.js

```diff
@@ -298,6 +298,12 @@ Example: $0 --help run.
         type: 'string',
         coerce: coerceCLICustomPreference,
       },
+      'start-url': {
+        describe: 'Launch firefox at specified page',
```

Comment:
@saintsebastian do you mind to add some alias here, e.g. based on what we discussed with @gregglind in #647:

```
'start-url: {
  alias: ['u', 'url'],
  ...
}
```

so that we can support a shorter cli syntax (helpful especially if the user wants to pass more than one url), while we keep the more descriptive name internally (which helps to make the internals more readable).